src/components/Tabs/TabPanel.js

```diff
@@ -0,0 +1,32 @@
+import { Grid } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+function TabPanel({ children, value, index, ...other }) {
+  return (
+    <Grid
+      container
+      direction="column"
+      role="tabpanel"
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
```

Comment:
1. Why `Grid`?
2. Aren't we setting `hidden` as well?

Reply:
@kilemensi Changed the above. For the hidden prop,  I did ran into an unusual behaviour where both all values in tabpanel were being shown whether or not the values were hidden. Removed it to test out why and forgot to add it again once i managed to fix it